spdlog/src/error.rs

```diff
@@ -102,6 +102,11 @@ pub enum Error {
     #[error("{0:?}")]
     Multiple(Vec<Error>),
 
+    /// Returned by [`Formatter`]s when an error occurs in serializing a log.
+    #[cfg(feature = "serde")]
+    #[error("failed to serialize log: {0}")]
+    Serialization(io::Error),
```

Comment:
```suggestion
    /// Returned by [`Formatter`]s when an error occurs in serializing a log.
    #[cfg(feature = "serde")]
    #[error("failed to serialize record: {0}")]
    SerializeRecord(io::Error),
```

For consistent naming style.